portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,61 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+h1 {
+  display: flex;
+  justify-content: center;
+}
+
+.icon-row {
+  width: 100%;
+  text-align: center;
+  overflow: auto;
```

Comment:
it's good practice to sort the styles within a class alphabetically.  you should do that here and for all the other classes on the page

Reply:
i don't think it's done for all of the styles?